hawk/src/main/java/com/orhanobut/hawk/Hawk.java

```diff
@@ -175,28 +178,39 @@ public void run() {
     //if any exception occurs during encoding, encodedText will be null and thus operation is unsuccessful
     return encodedText != null && storage.put(key, encodedText);
   }
-  //
-  //  /**
-  //   * Saves the list of objects to the storage
-  //   *
-  //   * @param key   is used to save the data
-  //   * @param value is the data that will be saved
-  //   * @return true if put is successful
-  //   */
-  //  @Deprecated
-  //  public static <T> boolean put(String key, List<T> value) {
-  //    if (key == null) {
-  //      throw new NullPointerException("Key cannot be null");
-  //    }
-  //    //if the value is null, simply remove it
-  //    if (value == null) {
-  //      return remove(key);
-  //    }
-  //
-  //    String encodedText = zip(value);
-  //    //if any exception occurs during encoding, encodedText will be null and thus operation is unsuccessful
-  //    return encodedText != null && storage.put(key, encodedText);
-  //  }
+
+  /**
+   * Creates a stream to put data
```

Comment:
in java doc a warning will be useful to say this function requires rxandroid library and throws null pointer exception if not